.gitignore

```diff
@@ -0,0 +1,2 @@
+.doesnotwork
```

Comment:
I love how you managed to fix all our bugs, as well as the features that don't work in one simple line.

Genius.